it/uk/gov/hmrc/fhregistrationfrontend/controllers/BusinessPartnersConfirmAddressControllerISpec.scala

```diff
@@ -9,50 +10,52 @@ import uk.gov.hmrc.fhregistrationfrontend.testsupport.{Specifications, TestConfi
 class BusinessPartnersConfirmAddressControllerISpec
   extends Specifications with TestConfiguration {
 
-  val route = "/business-partners/confirm-partner-address"
+  def route(mode: Mode): String = routes.BusinessPartnersConfirmAddressController.load(1, mode).url.drop(6)
 
-  s"GET $route" when {
+  List(NormalMode, CheckMode).foreach{ mode =>
+    s"GET $baseUrl${route(mode)}" when {
 
-    "the new business partners flow is enabled" should {
+      "the new business partners flow is enabled" should {
 
-      "render the business partner confirm address page" when {
-        "the user is authenticated" in {
-          given.commonPrecondition
+        "render the business partner confirm address page" when {
+          "the user is authenticated" in {
```

Comment:
Once using the new action you will need to set useranswers i.e           addUserAnswersToSession(emptyUserAnswers)